gen/benchmark_test.go

```diff
@@ -9,13 +9,16 @@ import (
 	tc "go.uber.org/thriftrw/gen/internal/tests/containers"
 	ts "go.uber.org/thriftrw/gen/internal/tests/structs"
 	"go.uber.org/thriftrw/protocol"
+	"go.uber.org/thriftrw/protocol/binary"
+	"go.uber.org/thriftrw/protocol/stream"
 	"go.uber.org/thriftrw/ptr"
 	"go.uber.org/thriftrw/wire"
 )
 
 type thriftType interface {
 	ToWire() (wire.Value, error)
 	FromWire(wire.Value) error
+	Encode(stream.Writer) error
```

Comment:
Should be in a separate interface declaration - like you currently have in `gen/utils_for_test.go`